src/fragments/lib/push-notifications/common/testing.mdx

```diff
@@ -0,0 +1 @@
+You can create messaging campaigns and send push notifications to your app with Amazon Pinpoint! Just follow these instructions on [Amazon Pinpoint User Guide](https://docs.aws.amazon.com/pinpoint/latest/userguide/messages-mobile.html) for the next steps.
```

Comment:
Shouldn't we have the steps mentioned specifically on our docs on how to send the Push Notification message from the Pinpoint console?

Reply:
@abdallahshaban557 I think that would create an extra spot for us to update if the pinpoint instructions change?

Reply:
I think it is worth it though - the pinpoint experience is really suboptimal - and it will be something that backend admins/product managers/marketers interact with a lot.

Reply:
I feel like it's better to link the pinpoint user guide for this since it's functionality of the console/service, not the library. Also, the linked page already provides nice instructions for that and if we were to duplicated it we would need a way to keep it in sync when they update their interface.

Reply:
understood - resolving this